### PR TITLE
SYS-3941 add log lowered claimed event

### PR DESCRIPTION
**Pull request by MBrozhko34**
## Proposed changes

Added LogLoweredClaimed event to primitives, add event handling and update token manager to consume the event.

## Type of change/Merge

🚨What type of change is this PR? </br>
_Put an `x` in the boxes that apply_

- [ ] Release 
  - [ ] Increase versions 
  - [x] Baseline tests passed  
  - Release type:
    - [ ] Major release 
    - [ ] Minor release 
    - [ ] Patch release 

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR._

- [x] You describe the purpose of the PR, e.g.:
  - What does it do?
  - Highlight what important points reviewers should know about;
  - Indicates if there is something left for follow-up PRs.
- [ ] Documentation updated
- [ ] Business logic tested successfully
- [ ] Verify First, Write Last: In Substrate development, it is important that you always ensure preconditions are met and return errors at the beginning. After these checks have completed, then you may begin the function's computation.

### CODE REVIEW

**Comment by MBrozhko34** [comment]
TO-DO: Token Manager tests

**Comment by MBrozhko34** [comment]
TO-DO: token-manager tests

File: pallets/token-manager/src/lib.rs [hunk 1]
```diff
@@ -976,7 +989,14 @@ impl<T: Config> Pallet<T> {
...
     fn on_event_processed(event: &EthEvent) -> DispatchResult {
-        return Self::lift(event)
+        return match &event.event_data {
+            EventData::LogLifted(d) => return Self::process_lift(event, d),
+            EventData::LogAvtGrowthLifted(d) => return Self::process_avt_growth_lift(event, d),
+            EventData::LogLowerClaimed(d) => return Self::process_lower_claim(event, d),
+
+            // Event handled or it is not for us, in which case ignore it.
+            _ => Ok(()),
+        }
```
Comment (alexpArtos) [inline pallets/token-manager/src/lib.rs:999]: :100:

**Review by alexpArtos — APPROVED** [review]

**Review by ivan-cholakov — APPROVED** [review]

File: pallets/token-manager/src/test_avt_tokens.rs [hunk 2]
```diff
@@ -341,3 +396,47 @@ fn avn_test_avt_token_total_lowered_amount_greater_than_balance_max_value_ok() {
...
+        let mock_event = &mock_data.lower_claimed_event;
+        insert_to_mock_processed_events(&mock_event.event_id);
+
+        let lower_id = 0;
+
+        perform_lower_setup(lower_id);
+
+        assert_ok!(TokenManager::on_event_processed(&mock_event));
+
+        assert!(System::events().iter().any(|a| a.event ==
```
Comment (alexpArtos) [inline pallets/token-manager/src/test_avt_tokens.rs:418]: might as well add that the lower is not present in the storage item ReadyToClaim

**Review by alexpArtos — APPROVED** [review]

**Pull request merged**